src/pages/index.js

```diff
@@ -141,6 +145,9 @@ const modalFormImage = new PopupWithForm({
       })
       .finally(() => {
         modalFormImage.renderLoading(false);
+        if ("add-card-form" in formValidators) {
+          formValidators["add-card-form"].resetValidation();
+        }
```

Comment:
`resetValidation` is needed only when you open modals. Please, remove them from `finally`.